reference/docs-conceptual/whats-new/differences-from-windows-powershell.md

```diff
@@ -474,8 +609,148 @@ For example, the following two strings are considered to be identical:
 True
 ```
 
+## New cmdlets
+
+### New Get-Uptime cmdlet
+
+The [Get-Uptime][Get-Uptime] cmdlet returns the time elapsed since the last boot of the operating
+system. The cmdlet was introduced in PowerShell 6.0.
+
+### New Remove-Alias cmdlet
+
+The [Remove-Alias][Remove-Alias] cmdlet removes an alias from the current PowerShell session. The
+cmdlet was introduced in PowerShell 6.0.
+
+### New cmdlet Remove-Service
+
+The `Remove-Service` cmdlet removes a Windows service in the registry and in the service database.
+The `Remove-Service` cmdlet was introduced in PowerShell 6.0.
+
+### New Markdown cmdlets
+
+Markdown is a standard for creating readable plaintext documents with basic formatting that can be
+rendered into HTML.
+
+The following cmdlets were added in PowerShell 6.1:
+
+- [ConvertFrom-Markdown][ConvertFrom-Markdown] - Convert the contents of a string or a file to a
+  **MarkdownInfo** object.
+- [Get-MarkdownOption][Get-MarkdownOption] - Returns the current colors and styles used for
+  rendering Markdown content in the console.
+- [Set-MarkdownOption][Set-MarkdownOption] - Sets the colors and styles used for rendering Markdown
+  content in the console.
+- [Show-Markdown][Show-Markdown] - Displays Markdown content in the console or as HTML
+
+### New Test-Json cmdlet
+
+The [Test-Json] cmdlet tests whether a string is a valid JavaScript Object Notation (JSON) document
+and can optionally verify that JSON document against a provided schema.
+
+This cmdlet was introduced in PowerShell 6.1
+
+### New cmdlets to support Experimental Features
+
+The following cmdlets were added in PowerShell 6.2 to support Experimental Features.
+
+- [Disable-ExperimentalFeature][Disable-ExperimentalFeature]
+- [Enable-ExperimentalFeature][Enable-ExperimentalFeature]
+- [Get-ExperimentalFeature][Get-ExperimentalFeature]
+
+### New Join-String cmdlet
+
+The [Join-String] cmdlet combines objects from the pipeline into a single string. This cmdlet was
+added in PowerShell 6.2.
+
+### New view ConciseView and cmdlet Get-Error
+
+PowerShell 7.0 enhances the display of error messages to improve the readability of interactive and
+script errors with a new default view **ConciseView**. The views are user-selectable through the
```

Comment:
Not _positive_ on the grammar for this one, but I think part of the struggle is the rhythm of `view ConciseView` when I read it aloud/subvocalize.

Not sure if we should specify `default error view` here - that's more explicit but also contextually clear _to me_.

```suggestion
script errors with a new default view, **ConciseView**. The views are user-selectable through the
```